pkg/build/builder/sti.go

```diff
@@ -259,7 +264,11 @@ func (s *S2IBuilder) Build() error {
 		return errors.New(buffer.String())
 	}
 
-	glog.V(4).Infof("Creating a new S2I builder with build config: %#v\n", describe.Config(config))
+	client, err := docker.NewEngineAPIClient(config.DockerConfig)
```

Comment:
A comment somehow connecting the dots to the docker daemon load issue might be useful months from now when someone looks at this and asks why we don't construct the client just once.